extensions/jfr/runtime/src/main/java/io/quarkus/jfr/runtime/JfrInfrastructure.java

```diff
@@ -0,0 +1,39 @@
+package io.quarkus.jfr.runtime;
+
+import org.jboss.logging.Logger;
+
+import io.quarkus.jfr.runtime.config.JfrRuntimeConfig;
+import io.quarkus.jfr.runtime.http.rest.RestEndEvent;
+import io.quarkus.jfr.runtime.http.rest.RestPeriodEvent;
+import io.quarkus.jfr.runtime.http.rest.RestStartEvent;
+import io.quarkus.runtime.annotations.Recorder;
+import jdk.jfr.FlightRecorder;
+
+@Recorder
+public class JfrInfrastructure {
```

Comment:
Very minor: can we call this `JfrRecorder?`

Reply:
I have changed this name to `JfrRecorder`.